app/src/main/java/helium314/keyboard/keyboard/internal/keyboard_parser/floris/KeyCode.kt

```diff
@@ -131,6 +131,9 @@ object KeyCode {
     const val ACTION_PREVIOUS =           -10007
     // Code value representing the code is not specified.
     const val NOT_SPECIFIED =             -10008 // todo: not sure if there is need to have the "old" unspecified keyCode different, just test it and maybe merge
+    const val CLIPBOARD_COPY_ALL =        -10009
+    const val MOVE_PAGE_UP =              -10010
+    const val MOVE_PAGE_DOWN =            -10011
```

Comment:
I think the keys codes should be names just `PAGE_UP` / `PAGE_DOWN`. Naming is not fully consistent, but I'm trying to...

Reply:
i agree, it will be changed